protocol/contracts/leaser/src/msg.rs

```diff
@@ -79,6 +84,20 @@ pub enum SudoMsg {
         lease_position_spec: PositionSpecDTO,
         lease_due_period: Duration,
     },
+    CloseProtocol {
+        migration_spec: <HigherOrderGranularity<HigherOrderProtocolContracts, HigherOrderOption> as HigherOrderType>::Of<MigrationSpec>,
+        #[serde(default)]
+        force: ForceClose,
+    }
+}
+
+#[derive(Serialize, Deserialize, Default, Clone, PartialEq, Eq, JsonSchema)]
+#[cfg_attr(any(test, feature = "testing"), derive(Debug))]
+#[serde(deny_unknown_fields, rename_all = "snake_case")]
+pub enum ForceClose {
+    #[default]
+    No,
+    KillProtocol,
```

Comment:
With this suggestion the API wouldn't change overall whilst still not including this variant on any other networks than the explicitly opt-in.
```suggestion
    #[cfg(any(feature = "net_dev", feature = "net_test"))]
    KillProtocol,
```